docs/en/operations/storing-data.md

```diff
@@ -213,9 +213,9 @@ Cache **commands**:
 
 - `SYSTEM DROP FILESYSTEM CACHE (<path>) (ON CLUSTER)`
 
-- `SHOW CACHES` -- show list of caches which were configured on the server.
+- `SHOW FILESYSTEM CACHES` -- show list of caches which were configured on the server. (For versions <= `22.8` commands is named `SHOW CACHES`)
```

Comment:
In my local CH, `SHOW CACHES` returns an empty result so I can't reakkt tell what it *should* return. Some remarks:

- `SHOW CACHES` / `SHOW FILESYSTEM CACHES` should be documented on https://clickhouse.com/docs/en/sql-reference/statements/show

- Likewise, https://clickhouse.com/docs/en/operations/caches/ could also mention the filesystem cache.

- If I read this PR correctly, the previous statement `SHOW CACHES` *only* referred to the filesystem cache(s). In that case, the previous naming was indeed misleading. Just wondering if there is a `SHOW` command for other cache types, e.g. the regexp cache?

- About documentation again. L. 214 is about dropping the filesystem cache. Could you please add that also to https://clickhouse.com/docs/en/sql-reference/statements/system?

- L. 216 should say "show list of filesystem caches [...]" instead of "show list of caches".

- Minor: (l. 216) "commands is names" --> "the command is named"

- About l. 218, it would maybe be cool to document DESCRIBE CACHE in https://clickhouse.com/docs/en/sql-reference/statements/describe-table.

Reply:
btw not sure if it these cache commands should be documented an show/, describe/, system/ sections, because they are documented on the page devoted to filesystem caches and there is a description for all of them. As a result there will be 2 places to update and keep in sync, as a result they might diverge...

> Likewise, https://clickhouse.com/docs/en/operations/caches/ could also mention the filesystem cache.

it already does...

> Just wondering if there is a SHOW command for other cache types, e.g. the regexp cache?

No, there isn't.

Reply:
Yepp. Having a full description in just a single place is a good idea.